core/src/test/java/kafka/test/junit/ClusterTestExtensions.java

```diff
@@ -123,8 +123,7 @@ private void processClusterTemplate(ExtensionContext context, ClusterTemplate an
         if (!annot.value().isEmpty()) {
             generateClusterConfigurations(context, annot.value(), generatedClusterConfigs::add);
         } else {
-            // Ensure we have at least one cluster config
-            generatedClusterConfigs.add(ClusterConfig.defaultClusterBuilder().build());
+            throw new IllegalStateException("Annotation value can't be empty string.");
```

Comment:
Could we check `annot.value().isEmpty()` first? for example:
```java
if (annot.value().isEmpty()) throw exception;
```
With above change, we can simplify the if-else